predict_on_directory.py

```diff
@@ -0,0 +1,35 @@
+########################################################################################################################
+# Model
+########################################################################################################################
+
+import numpy as np
+from classifiers import *
+from pipeline import *
+
+import os
+import glob
+import sys
+from tensorflow.keras.preprocessing.image import load_img, img_to_array
+
+REQUIRED_SIZE = (256, 256)
+
+if __name__ == "__main__":
+    images_dir = sys.argv[1]
+
+    if not os.path.isdir(images_dir):
+        print("## Directory provided {} doesn't exist.".format(images_dir))
+        exit()
+
+    # 1 - Load the model and its pretrained weights
+    classifier = Meso4()
+    classifier.load('weights/Meso4_DF')
```

Comment:
Well, for the purpose of a command line script, this needs parametrisation, what if your directory is forged using face-2-face.